apps/src/code-studio/peer_reviews/PeerReviewSubmissionData.story.jsx

```diff
@@ -61,9 +55,7 @@ export default storybook => {
                 submission_date: '2/19/2017',
                 escalation_date: undefined,
                 review_ids: [[1, ''], [2, 'accepted']],
-                status: 'escalated',
-                accepted_reviews: 1,
-                rejected_reviews: 0
+                status: 'escalated'
```

Comment:
How do we get in a state where the `status` is 'escalated' but `escalation_date` is undefined?

Reply:
I'm not sure - this may not be a representative story.  I'll review.